### ISSUE TITLE
[`WRONG_INDENTATION`] false positive warning on functions without body

### ISSUE THREAD

**Issue by WebTiger89** [saveourtool/diktat#1451]
I'm not sure if this case was already covered:

```kotlin
fun test(
    flag: Boolean,
): String =
    with(flag) {
        ""
    }

or

fun simpleFun() =
    "Wrong indentation"

```

When `extendedIndentAfterOperators: true` the above code get false positive warnings. If `extendedIndentAfterOperators: false` the warnings disappear but will appear for other pieces of code like:

```kotlin
fun test() {
    println("This is a long comment " +
            "that requires multiple lines.")
}
```

**Comment by 0x6675636b796f75676974687562** [comment]
@WebTiger89, you're right indeed.

That's why, as of #1443, we've introduced two separate flags (`extendedIndentAfterOperators` for expressions and `extendedIndentForExpressionBodies` for expression-body functions), consistently with IDEA. The change will make it into the **1.2.2** release, or you can build and try the snapshot version yourself.

We're working to improve our compatibility with Kotlin style guide (as implemented by IDEA), see [this wiki page](https://github.com/saveourtool/diktat/wiki/Mapping-of-flags-which-affect-indentation) for details.

I'm closing this issue as a duplicate of #1443.

If you believe this is not true, feel free to reopen.
